test.md

```diff
@@ -0,0 +1,318 @@
+---
+description: Smoketest page
```

Comment:
I think if we leave this out of the nav and exclude it from the sitemap by adding `hide_from_sitemap: true` to the front-matter, that's probably enough.